packages/amplication-data-service-generator/src/server/resource/controller/create-controller.ts

```diff
@@ -173,6 +175,52 @@ async function createControllerModule(
 
     classDeclaration.body.body.push(...toManyRelationMethods);
 
+    setEndpointPermissions(
+      classDeclaration,
+      mapping["CREATE_ENTITY_FUNCTION"] as namedTypes.Identifier,
+      EnumEntityAction.Create,
+      entity
+    );
+    setEndpointPermissions(
+      classDeclaration,
+      mapping["FIND_MANY_ENTITY_FUNCTION"] as namedTypes.Identifier,
+      EnumEntityAction.Search,
+      entity
+    );
+    setEndpointPermissions(
+      classDeclaration,
+      mapping["FIND_ONE_ENTITY_FUNCTION"] as namedTypes.Identifier,
+      EnumEntityAction.View,
+      entity
+    );
+    setEndpointPermissions(
+      classDeclaration,
+      mapping["UPDATE_ENTITY_FUNCTION"] as namedTypes.Identifier,
+      EnumEntityAction.Update,
+      entity
+    );
+    setEndpointPermissions(
+      classDeclaration,
+      mapping["DELETE_ENTITY_FUNCTION"] as namedTypes.Identifier,
+      EnumEntityAction.Delete,
+      entity
+    );
+
+    toManyRelationFields.map((field) => {
+      const toManyFunctionMapping = {
+        FIND_MANY: builders.identifier(camelCase(`findMany ${field.name}`)),
+        CONNECT: builders.identifier(camelCase(`connect ${field.name}`)),
+        DISCONNECT: builders.identifier(camelCase(`disconnect ${field.name}`)),
+        UPDATE: builders.identifier(camelCase(`update ${field.name}`)),
+      };
+
+      return setToManyEndpointPermission(
+        classDeclaration,
+        toManyFunctionMapping,
+        entity
+      );
+    });
```

Comment:
I think this part should be included in `createToManyRelationMethods` - it will save another iteration on all the relation fields and creation of all the identifiers